src/ilcdlib/entity/compliance.py

```diff
@@ -0,0 +1,117 @@
+#
+#  Copyright 2023 by C Change Labs Inc. www.c-change-labs.com
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#  This software was developed with support from the Skanska USA,
+#  Charles Pankow Foundation, Microsoft Sustainability Fund, Interface, MKA Foundation, and others.
+#  Find out more at www.BuildingTransparency.org
+#
+from typing import Type
+
+from openepd.model.standard import Standard
+
+from ilcdlib.common import (
+    BaseIlcdMediumSpecificReader,
+    IlcdXmlReader,
+    OpenEpdComplianceSupportReader,
+    OpenEpdStandardReader,
+)
+from ilcdlib.dto import IlcdReference
+from ilcdlib.type import LangDef
+from ilcdlib.utils import none_throws
+from ilcdlib.xml_parser import T_ET
+
+
+class IlcdStandardReader(OpenEpdStandardReader, IlcdXmlReader):
```

Comment:
In fact we read ILCD's `Source` object which might define multiple things including PCRs and Standards. I think it is time to move reading name, version, ref, uuid and other things into `IlcdSourceReader` and make it a parent for `IlcdStandardReader` and `IlcdPcrReader`